changelog.d/CaptureAll

```diff
@@ -0,0 +1,14 @@
+synopsis: Bugfix - CaptureAll produces [""] for empty paths due to trailing slash.
+prs: #1516
+issues: #1243
+
+description: {
+
+CaptrueAll resulted in `[""]` for empty paths due to trailing slash.  Similar
```

Comment:
`CaptureAll` *

Reply:
Thanks you.  I'll get the typo fixed this afternoon EST.

Reply:
Oh, I forgot to get to this.  I'll get it today.

Reply:
I amended and force pushed.  Apologies to anyone who might have fetched my branch.